website/docs/getting-started/using-the-docker_host-environment-variable.md

```diff
@@ -0,0 +1,39 @@
+---
+id: using-the-docker_host-environment-variable
+title: Using the `DOCKER_HOST` environment variable
+description: Using the `DOCKER_HOST` environment variable can make it easier to migrate from Docker to Podman Desktop, as it allows you to continue using familiar Docker commands while taking advantage of the benefits of Podman.
+tags: [podman-desktop, getting-started]
+keywords: [podman desktop, podman, containers, docker_host, environment, variable]
+hide_table_of_contents: false
+---
+
+# Using the `DOCKER_HOST` environment variable
+
+Consider using the `DOCKER_HOST` environment variable to migrate transparently from Docker to Podman Desktop on all platforms.
+
+* Continue using familiar Docker commands.
+* Take advantage of the benefits of Podman.
+* Your tools, such as [Gradle](https://gradle.org/) or [Testcontainers](https://www.testcontainers.org/), communicate with Podman without reconfiguration.
+
+#### Prerequisites
+
+* Podman
+* [jq](https://stedolan.github.io/jq/)
+
+#### Procedure
+
+1. Identify the location of your Podman socket:
+
+    ```shell-session
+    $ podman info --format json | jq '.host.remoteSocket.path'
+    ```
```

Comment:
it'll display the socket of Podman engine but not the socket on the host
so for example on macOS or Windows it'll show the path inside the VM which won't work when connecting from the host

using format flag, you don't need an external tool like jq (that probably many people don't have)

On Windows and macOS the command should be:
```
podman machine inspect --format '{{.ConnectionInfo.PodmanSocket.Path}}'
/Users/benoitf/.local/share/containers/podman/machine/podman-machine-default/podman.sock
```

Reply:
But this command does'nt work on Linux, so we don't have a generic command for all platforms, correct?

Reply:
correct, it depends if we have a VM or not